lang/test/org/partiql/lang/syntax/SqlParserTestBase.kt

```diff
@@ -86,9 +86,9 @@ abstract class SqlParserTestBase : TestBase() {
     ) {
         // Check for V0 Ast
         val actualExprNode = parse(source)
-        val expectedV0Ast = loadIonSexp(expectedV0Ast)
+        val expectedV0AstSexp = loadIonSexp(expectedV0Ast)
 
-        serializeAssert(AstVersion.V0, actualExprNode, expectedV0Ast, source)
+        serializeAssert(AstVersion.V0, actualExprNode, expectedV0AstSexp, source)
```

Comment:
For the comment https://github.com/partiql/partiql-lang-kotlin/pull/462#discussion_r739550495.